app/test/unit/stores/updates/update-conflict-state-test.ts

```diff
@@ -7,14 +7,19 @@ import {
   ManualConflictResolution,
   ManualConflictResolutionKind,
 } from '../../../../src/models/manual-conflict-resolution'
+import { IStatsStore } from '../../../../src/lib/stats'
 
 describe('updateConflictState', () => {
-  const statsStore = {
-    recordMergeAbortedAfterConflicts: jest.fn(),
-    recordMergeSuccessAfterConflicts: jest.fn(),
-    recordRebaseAbortedAfterConflicts: jest.fn(),
-    recordRebaseSuccessAfterConflicts: jest.fn(),
-  }
+  let statsStore: IStatsStore
+  beforeEach(() => {
+    statsStore = {
```

Comment:
The original change in this test indicated that `statsStore.recordRebaseAbortedAfterConflicts` had been called twice (likely by other tests), so it would fail. This change is to ensure the mocks are not reused across tests, and we could explicitly assert here that other mocks were not called, but I felt that would make the tests overly verbose.